core/src/main/java/com/novoda/noplayer/mediaplayer/AndroidMediaPlayerImpl.java

```diff
@@ -264,11 +267,37 @@ public void selectAudioTrack(PlayerAudioTrack audioTrack) {
     }
 
     @Override
+    public boolean hasAvailableSubtitles() {
+        NoPlayerLog.w("Subtitles not implemented for Android Media Player");
```

Comment:
Maybe a warning is enough but I would maybe make sure there is a `Unit` test written for this even if they are dummy tests for now. For example here asserting we return false, or in the others verifying no interaction with the input arguments

The reason is that we devoted a good time writing test for this and it is proven that we might easily forget about maintaining them. What do you think guys?